pkgs/tools/system/netdata/default.nix

```diff
@@ -37,7 +37,7 @@ in stdenv.mkDerivation rec {
     ++ optionals stdenv.isDarwin [ CoreFoundation IOKit libossp_uuid ]
     ++ optionals (!stdenv.isDarwin) [ libcap.dev libuuid.dev ]
     ++ optionals withCups [ cups ]
-    ++ optionals withDBengine [ judy lz4.dev ]
+    ++ optionals withDBengine [ lz4 ]
```

Comment:
Is the `lz4.dev` -> `lz4` change related to this? (If not, I'm confused as to why it's suddenly not needed.)

Reply:
I believe it shouldn't be needed in the first place.

Reply:
According to the build log, lz4 is still correctly found.
```
netdata> checking for LZ4_initStream in -llz4... yes
netdata> checking for LZ4_compress_default in -llz4... yes
```

Reply:
Wonder if it's required for `libcap` and `libuuid`, you'd think not. Hm.

Reply:
stdenv takes care of selecting the correct output most of the times.

Reply:
Yes, which is why I think if we're cleaning it up for one, might as well do it for all of them in this PR. Maybe a separate commit, though -- not sure.